### PR TITLE
Add auth system

### PR DESCRIPTION
**Pull request by nerdylive123**

### CODE REVIEW

**Comment by nerdylive123** [comment]
Alright waiting thanks

File: server/app.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-const express = require('express')
-const app = express()
-const port = 3000
+const express = require('express'),
+    app = express();
```
Comment (jxpsert) [inline server/app.js:2]: Why did you remove ``const`` here?

File: server/routes/auth.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const express = require('express'),
+    router = express.Router();
```
Comment (jxpsert) [inline server/routes/auth.js:2]: Same here, why no ``const``?

**Review by jxpsert — CHANGES_REQUESTED** [review]
2 comments

File: server/app.js (hunk 1 above)
Comment (nerdylive123) [reply server/app.js:2]: its the same

File: server/routes/auth.js (hunk 2 above)
Comment (nerdylive123) [reply server/routes/auth.js:2]: the same, shorter and easier to read

**Review by jxpsert — APPROVED** [review]
alrighty, looks fine

**Pull request merged**